release/x86_64-freebsd10/Vagrantfile

```diff
@@ -81,6 +81,9 @@ Vagrant.configure(2) do |config|
   #   sudo apt-get install -y apache2
   # SHELL
   config.vm.provision "shell", inline: <<-SHELL
-    pkg install -y bash gmake ruby rubygem-bundler git-subversion python2
+    freebsd-update --not-running-from-cron fetch install
```

Comment:
Doesn't this get automatically run when you first start the box? I thought the box itself had an auto-upgrade script that was run on first boot?